1-getting-started-lessons/translations/README.fr.md

```diff
@@ -1,16 +1,16 @@
 # Premiers pas avec le développement Web
 
-Dans cette section du programme, vous serez initié à des concepts non basés sur des projets importants pour devenir un développeur professionnel.
+Dans cette section du programme, vous serez initié à des concepts importants pour devenir un développeur professionnel non basés sur des projets.
```

Comment:
this is an awkward sentence no matter what, but I'll approve it

Reply:
Thank you @jlooper